contrib/pkg/clusterpool/clusterclaim.go

```diff
@@ -0,0 +1,106 @@
+package clusterpool
+
+import (
+	"fmt"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
```

Comment:
Separate the k8s imports.
```suggestion
	log "github.com/sirupsen/logrus"
	"github.com/spf13/cobra"
	
	"k8s.io/apimachinery/pkg/api/meta"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	"k8s.io/client-go/kubernetes/scheme"
```